language/move-lang/README.md

```diff
@@ -97,7 +97,7 @@ FLAGS:
 OPTIONS:
     -s, --sender <address>                             The sender address for modules and scripts
     -d, --dependencies <path-to-dependency-file>...    The library files needed as dependencies
-    -o, --out-dir <path-to-output-directory>           The directory for outputing move bytecode [default: output]
+    -o, --out-dir <path-to-output-directory>           The move bytecode output directory [default: move_build_output]
```

Comment:
I know this isn't part of your change, but since you're touching this anyway, can you capitalize "Move bytecode"?